File: 1915-check-if-one-string-swap-can-make-strings-equal/1915-check-if-one-string-swap-can-make-strings-equal.py
class Solution:
    def areAlmostEqual(self, s1: str, s2: str) -> bool:
        S2=sorted(s2)
        S1=sorted(s1)
        if(S1!=S2):
            return False
        count=0
        for i in range(len(s1)):
            if s1[i]!=s2[i]:
                count+=1
        if count==0 or count ==2:
            return True
        return False
        

File: 1915-check-if-one-string-swap-can-make-strings-equal/1915-check-if-one-string-swap-can-make-strings-equal.cpp
class Solution {
public:
    bool areAlmostEqual(string s1, string s2) {
        int count=0;
        string S1=s1;
        string S2=s2;
        sort(S1.begin(),S1.end());
        sort(S2.begin(),S2.end());
        if(S1!=S2) return false;
        for(int i=0;i<s2.size();i++ ){
            if(s1[i]!=s2[i]) count++;
        }
        if(count==0 || count==2) return true;
        return false;
     
        
    }
};